include/flatbuffers/reflection.h

```diff
@@ -243,7 +255,28 @@ template<typename T> T *GetAnyFieldAddressOf(const Struct &st,
 template<typename T> bool SetField(Table *table, const reflection::Field &field,
                                    T val) {
   assert(sizeof(T) == GetTypeSize(field.type()->base_type()));
-  return table->SetField(field.offset(), val);
+  T def;
+  switch (field.type()->base_type()) {
```

Comment:
Nevermind, just realized those enum values are generated from the reflection.fbs, and if we're following the normal schema upgrade conventions those shouldnt be changed.

I'll update PR.